optlib/puppetManifest.ctags

```diff
@@ -39,6 +39,11 @@
 --kinddef-PuppetManifest=r,resource,resources
 --kinddef-PuppetManifest=v,variable,variables
 
+#
+# Separators
+#
+--_scopesep-PuppetManifest=c.,::
```

Comment:
It would be nice to generate the `::` prefix also for definitions. For example [here](https://github.com/ahakanbaba/ctags/blob/7bc5ac9ff642dab372a650faf151ae86105e1e3d/Units/parser-puppetManifest.r/puppet-classpathtest.d/input.pp#L3)
For that do we need to pass an additional `--_scopesep-PuppetManifest=d.,::` ?

Reply:
Now it should be `--_scopesep-PuppetManifest=d,:::`.
I changes letters used in the option. I would like not to use `.`. I would like to reserver it for future extension.

Reply:
See the documentation. 
x,: => root prefix for x
You can use wildcards `*`.